src/CSSBattle/2_Carrom/ZhangZhikai.html

```diff
@@ -0,0 +1,38 @@
+<div class="box" >
+</div>
+<div class="box topright">
+</div>
+<div class="box bottomright">
+</div>
+<div class="box bottomleft">
+</div>
+
+<style>
+  body {
+    background: #62374E
+  }
+  div {
+    width: 50px;
+    height: 50px;
+    position: fixed;
+    background: #fdc57b;
+  }
+  .box {
+   	left: 50px;
```

Comment:
Consider removing unnecessary line break
```suggestion
     left: 50px;
```

Reply:
50ed92b92dcb6d7fac9a9b6965067d8e1abde559